index.js

```diff
@@ -88,8 +89,6 @@ module.exports = async (target, options) => {
 				const windowsPath = await wslToWindowsPath(options.app);
 				options.app = windowsPath;
 			}
-
-			cliArguments.push(options.app);
```

Comment:
Why are you removing this?

Reply:
I moved it `if`, and mistake to remove it